_posts/events/2019-05-08---Binding-your-serverless-clients.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: single
+title: "2019-05-08 - Binding your serverless clients"
+date: 2019-05-08 18:00:00 +0000
+comments: true
+published: true
+categories: ["events"]
+tags: ["Events"]
+author: Tom Kerkhove
+---
+
+## Agenda
+
+### Binding your serverless clients
+
+Scaling your services to hundreds of instances without breaking a sweat is probably the main reason you want to develop a serverless solution. But what about connecting to services which aren't supported out of the box? Are you creating your own `HttpClient`, `SqlConnection` or connecting via some obscure third-party client?  
+
+During this session, I'll be covering how you can create your own Azure Function custom binding types.
+
+By using these custom binding types you can reuse them all over your solution, use some of the best practices for each binding type and make sure your Azure Functions can interact and scale to any service with ease!
+
+If you want to create a production-ready serverless solution, be sure to attend this session!
+
+<img src="/assets/media/speakers/jan-de-vries.jpg" alt="Jan de Vries" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Jan de Vries has been developing software for well over a decade. 
+
+Currently, he is working at 4DotNet as a Cloud Solution Architect/Developer and is awarded by Microsoft with the Microsoft MVP award.
+
+His main focus is on developing highly performant and scalable solutions using the awesome services provided by the Microsoft Azure platform. Because of his expertise, he has been able to help out multiple customers to bring their on-premise solution to the cloud and guide them towards a better software development ecosystem. 
+
+Whenever Jan finds something interesting to write about, he shares his findings on [his blog](https://jan-v.nl).*
+
+### Slot II - To be confirmed
+To be confirmed
+
+## Practical details
+
+**Event date:** May 8, 2019 - you are welcome from 18:00, session starts 18:30
+
+**Event location:**<br />
+To be confirmed
+
+## Register via Pretix
+Thank you Maarten!
```

Comment:
Is this a hint to set up Pretix already?

Reply:
Yes but now that I think of it we should wait a while